src/Tools/DynamoShapeManager/Utilities.cs

```diff
@@ -92,6 +92,50 @@ private static List<string> ProductsWithASM
             "ADPSDKUI.DLL",
             "ADPSDKCORE.DLL"
```

Comment:
Do we want to continue preloading ADP dll's for ASM228?

Reply:
hmm, good question - I will file a followup to look into this as we'd have to make changes on the LibG side as well.